__tests__/app.test.js

```diff
@@ -29,6 +29,70 @@ describe("/api/", () => {
     });
   });
   describe("/api/reviews", () => {
+    describe('"/api/reviews', () => {
+      test("200: responds with all reviews if no query is set in descending order", () => {
+        return request(app)
+          .get("/api/reviews")
+          .expect(200)
+          .then((reviews) => {
+            const body = reviews.body;
+
+            const keys = [
+              "review_id",
+              "title",
+              "category",
+              "designer",
+              "owner",
+              "review_body",
+              "review_img_url",
+              "created_at",
+              "votes",
+              "comment_count",
+            ];
+            expect(body.length === 13).toBe(true);
+            body.forEach((review) => {
+              expect(Object.keys(review)).toEqual(keys);
+              expect(body[0].created_at > body[12].created_at).toBe(true);
+            });
+          });
+      });
+      test("200: responds with reviews matching a query in descending date order", () => {
+        return request(app)
+          .get("/api/reviews?category=social deduction")
```

Comment:
you're not testing that the reviews are `social deduction`!

make sure you are actually asserting against what your test description is saying!